src/communication/http/httpClient.ts

```diff
@@ -25,10 +25,10 @@ export abstract class HttpClient {
 
   public constructor(
     protected readonly logger: Logger,
-    baseUrl: string,
-    private readonly targetService = '',
-    retryConfig?: IHttpRetryConfig,
-    private readonly disableDebugLogs = false
+    protected baseUrl: string,
```

Comment:
is there anywhere url validator?

Reply:
i have opened an issue https://mapcolonies.atlassian.net/browse/MAPCO-2948